ysqlhelper/ysqlhelper.go

```diff
@@ -33,7 +33,7 @@ func PrepareTestContainer(t *testing.T, version string) (func(), string) {
 
 	runner, err := docker.NewServiceRunner(docker.RunOptions{
 		ImageRepo:     "yugabytedb/yugabyte",
-		Cmd:           []string{"./bin/yugabyted", "start", "--daemon=false"},
+		Cmd:           []string{"bash", "-c", "CONTAINER_IP=$(hostname -I | awk '{print $1}') && ./bin/yugabyted start --advertise_address=$CONTAINER_IP --background=false"},
```

Comment:
Will this work even for the earlier versions of YugabyteDB? Can you confirm through a quick test, if not already done?

Reply:
Yes, have tested with latest and 2024.1.4.0-b108 docker tags.